backend/src/test/java/ai/verta/modeldb/TestsInit.java

```diff
@@ -184,7 +192,7 @@ public static void setServerAndService() throws Exception {
 
   @AfterClass
   public static void removeServerAndService() throws InterruptedException {
-    App.initiateShutdown(0);
+    //    App.initiateShutdown(0);
```

Comment:
do we need this, or is this now done elsewhere?

Reply:
Removed, it is not used for now.